config/examples/redis.yml

```diff
@@ -1,22 +1,17 @@
-base: &base
+base: &default
   url: "<%= ENV.fetch('REDIS_URL', 'redis://localhost/1') %>"
   pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   pool_timeout: 5 # this is in seconds
   namespace: "<%= ENV['REDIS_NAMESPACE'] %>"
   sentinels: "<%= ENV['REDIS_SENTINEL_HOSTS'] %>"
   role: <%= ENV['REDIS_SENTINEL_ROLE'] %>
 
-production:
-  <<: *base
-  url: "<%= ENV.fetch('REDIS_URL', 'redis://localhost/3') %>"
-
-preview:
-  <<: *base
-  url: "<%= ENV.fetch('REDIS_URL', 'redis://localhost/3') %>"
-
 development:
-  <<: *base
+  <<: *default
 
 test:
-  <<: *base
+  <<: *default
   db: <%= 2 + ENV['TEST_ENV_NUMBER'].to_i %>
+
+production:
+  <<: *default
```

Comment:
Was it intentional to default production redis to `redis://localhost/1`? I think it makes sense to keep default different from `developement`.

Reply:
As I mentioned in [this comment](https://github.com/3scale/porta/pull/3225/files#r1123083017) I think having production environment is pretty useless for localhost in general, just when you want to test something which is production-specific. I don't think having a different DB will make much difference. But we can do it, just to follow the same pattern as for the relational DB.

Reply:
Actually, I'm more inclined to delete the default value in `production` and make the developer set the value explicitly.

Reply:
I think a good default is valuable. I had to start production locally a couple of times in the past. It's crap to need to touch many places instead of having things work mostly OOB

Reply:
In case we remove the default values, please @mayorova add any necessary extra steps to the docs

Reply:
I think it makes it more confusing if we reserve all different DB numbers for different environments. But I agree that it probably needs to be documented.